### ISSUE TITLE
Feature Request: Mailchimp Groups Support

### ISSUE THREAD

**Issue by calevans** [mautic/mautic#4360]
What type of report is this:

| Q  | A
| ---| ---
| Bug report? | 
| Feature request? | x 
| Enhancement? | 

## Description:
Mailchimp supports the concepts of groups. If I understand Mautic correctly, this is analogous to Segments. I would like to be able to set one or more groups when using the "Push to Integration" in Campaign Builder. 

Like the filed mapping, it would be nice to have a segment mapping.
  
Cheers! :)
=C=

**Comment by wucherpfennig** [comment]
+1

**Comment by dbhurley** [comment]
In order to improve engagement and increase visibility to this feature request we are closing this issue on GitHub and recommending all Feature Requests (without code attachment) be placed in the [Ideas forum](https://www.mautic.org/forums/forum/ideas) on our community site. 

We recommend you post your ideas to Mautic's forum so others can comment on them and help turn your ideas into a proof of concept and eventually an exciting new feature in Mautic. 

_More information about this move can be found in our [blog announcement](https://www.mautic.org/community-blog/moving-feature-requests/)._

**Comment by wucherpfennig** [comment]
This is bad. The forum is bad and almost dead...

**Comment by dbhurley** [comment]
I don't believe it's dead, or bad. :) _But_ the Issue tracker is for issues, not random feature suggestions. Therefore, adding ideas and getting feedback in the forum will also serve the additional purpose that if your supposed assumption of "bad and almost dead" were true - you'll be helping to fix that!

**Comment by wucherpfennig** [comment]
The forum lost half its old posts and is very confusing to work with.

And it seems that you mainly move things in order to look better on github...

**Comment by dbhurley** [comment]
It did not lose half it's posts, that's a bad exaggeration. Secondly, there's no confusion to it, in fact it's a very standard Buddypress installation :) 

You are absolutely right, we have improved GitHub now *AND* we have improved the opportunity for the features to be discussed. I am sorry if you are uncomfortable and unfamiliar with forum software. If you'd like to take this offline we can discuss it further and I can walk you through the basics of how to add/edit/update posts and things!
